backend/src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -23,6 +23,7 @@
   private final String time;
   private final String calories;
   private final String difficulty;
+  private final String image;
```

Comment:
nit: I think `image` is a bit of a weird name -- this String is actually the URL to the image, right? If that's the case I'd suggest something more descriptive like `imageUrl`